<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermianlRemotaCliente</class>
 <widget class="QMainWindow" name="TermianlRemotaCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TermianlRemotaCliente</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="txtIpCliente"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnConectar">
        <property name="text">
         <string>Conectar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDesconectar">
        <property name="text">
         <string>Desconectar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="tctComando"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnEnviar">
        <property name="text">
         <string>Enviar</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
